contracts/Origin/CertificateLogic.sol

```diff
@@ -144,12 +143,14 @@ contract CertificateLogic is CertificateInterface, RoleManagement, TradableEntit
         TradableEntityDBInterface(address(db)).setForSale(_certificateId, false);
     }
 
-    /// @notice creates a new Entity / certificate
+    /// @notice creates a new bundle
```

Comment:
Are bundles created in CertificateLogic? Aren't they created in EnergyCertificateBundleLogic?

Reply:
It should be `creates a new Entity / certificate`, because it doesn't create bundles